spec/lib/topic_query_spec.rb

```diff
@@ -20,6 +20,10 @@
   fab!(:moderator) { Fabricate(:moderator) }
   fab!(:admin) { Fabricate(:admin) }
 
+  before do
+    SiteSetting.enable_whispers = true
+  end
```

Comment:
Do we need this for all test cases? I feel like we should only enable it for tests that need it unlessmost tests need it here?